ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -222,68 +220,7 @@ class IdpLoginScreen extends React.Component<Props, State> {
       };
     }
     const idpTextData = selectedIdpTextData.value;
-    return {
-      title: I18n.t("authentication.idp_login.contextualHelpTitle2"),
-      body: () => (
-        <React.Fragment>
-          {/** Recover credentials */}
-          <Markdown>
-            {idpTextData.recover_username
-              ? I18n.t("authentication.idp_login.dualRecoverDescription")
-              : I18n.t("authentication.idp_login.recoverDescription")}
-          </Markdown>
-          <View spacer={true} />
-          {idpTextData.recover_username && (
-            <React.Fragment>
-              <BlockButtons
-                type={"SingleButton"}
-                leftButton={{
-                  title: I18n.t("authentication.idp_login.recoverUsername"),
-                  onPress: handleItemOnPress(idpTextData.recover_username),
-                  small: true
-                }}
-              />
-              <View spacer={true} />
-            </React.Fragment>
-          )}
-          <BlockButtons
-            type={"SingleButton"}
-            leftButton={{
-              title: I18n.t("authentication.idp_login.recoverPassword"),
-              onPress: handleItemOnPress(idpTextData.recover_password),
-              small: true
-            }}
-          />
-
-          {/** Idp cotnacts */}
-          <View spacer={true} />
-          <Markdown>{idpTextData.description}</Markdown>
-          <View spacer={true} />
-
-          <EmailCallCTA
-            phone={idpTextData.phone}
-            email={idpTextData.email ? idpTextData.email : undefined}
-          />
-          <View spacer={true} />
-
-          {idpTextData.helpdesk_form && (
-            <React.Fragment>
-              <BlockButtons
-                type={"SingleButton"}
-                leftButton={{
-                  title: I18n.t("authentication.idp_login.openTicket"),
-                  onPress: handleItemOnPress(idpTextData.helpdesk_form),
-                  primary: true,
-                  bordered: true,
-                  small: true
-                }}
-              />
-              <View spacer={true} />
-            </React.Fragment>
-          )}
-        </React.Fragment>
-      )
-    };
+    return IdpCustomContextualHelpContent({ idpTextData });
```

Comment:
```suggestion
    return IdpCustomContextualHelpContent(idpTextData);
```
This is because we should use `Idp` type instead of `any`

Reply:
it was because the function was accepting Props as input. Updated passing the idd as the function input only (https://github.com/pagopa/io-app/pull/1898/commits/732737daf8e67773abbeb723ee18a72f1004a450)